### DISCUSSION TITLE
[A-L] (2023/24) Foglio 1 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#132] [Esercizi - Bertini]
![Es 1](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/dfa28b88-c056-4295-af56-06989895c77c)

**Comment by daromacarlo** [comment]
1' punto)
![Immagine 2023-10-07 184315](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/2f1650c3-90d3-472e-a3e0-19fd225f6636)

non so se posso dimostrare così la prima relazione...  
Fatemi sapere, grazie.

**Comment by daromacarlo** [comment]
punto 2)
![Immagine 2023-10-07 192740](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/be0c5b07-4ade-4f29-b25a-70d405b0f6ca)

**Comment by daromacarlo** [comment]
Come dicevo, non sono sicuro che si possano dimostrare in maniera così discorsiva.
Quindi le ho dimostrate anche con la comparazione delle tavole di verità.

per esempio per la prima relazione avro:
![Immagine 2023-10-07 195407](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/ebeebe08-68ce-423a-b4e5-6239be639ac0)
e dato che le due tavole di verità sono uguali posso dire che la relazione è vera.

**Comment by FeddyLix17** [comment]
<div align="center">

<h1> 
    1. Proprietà distributiva dell'intersezione rispetto all'unione
 <br> $(A \cup  B) \cap C = (A \cap C) \cup (B \cap C)$ 
</h1>

</div>

$$
    \huge (A \cup  B) \cap C = \lbrace x\ |\  x \in C\ \wedge\ (x \in A\ \lor\ x \in B) \\rbrace
$$

$$
    \huge x \in A \implies x \in A \cap C
$$

$$
    \huge x \in C \implies x \in B \cap C
$$

In entrambi i casi $\huge x \in (A \cap C) \cup (B \cap C)$

---

<div align="center">

<h1> 
    2. Proprietà distributiva dell’unione rispetto all’intersezione <br> $(A \cap  B) \cup C = (A \cup C) \cap (B \cup C)$ 
</h1>

</div>

$$
    \huge (A \cap  B) \cup C = \lbrace x\ |\  (x \in A\ \wedge\ x \in B)\ \lor\ x \in C \\rbrace
$$

$$
    \huge x \in (A \cap B) \implies x \in (A \cup C)\ \wedge\ x \in (B \cup C)
$$

$$
    \huge x \in C \implies  x \in (C \cup A)\ \wedge\ x \in (C \cup B)
$$

In entrambi i casi $\huge x \in (A \cup C) \cap (B \cup C)$

---

<div align="center">

<h1> 
    3. Prima legge di De Morgan <br> $(A \cap  B)^C = A^C \cup B^C$ 
</h1>

</div>

$$
    \huge (A \cap  B)^C = \lbrace x\ |\  x \notin A\ \lor\ x \notin B \\rbrace
$$

$$
    \huge x \notin A \implies x \in A^C
$$

$$
    \huge x \notin B \implies x \in B^C
$$

In entrambi i casi $\huge x \in A^C \cup B^C$

**Comment by CasuFrost** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/97389762/f79eede3-61da-46c4-91be-affb41af920d)
